guides/user/kubernetes-v2/deploy-helm/index.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: single
+title:  "Deploy Helm Charts"
+sidebar:
+  nav: guides
+---
+
+{% include alpha version="1.8" %}
+
+{% include toc %}
+
+Spinnaker surfaces a "Bake (Manifest)" stage to turn templates into manifests
+with the help of a templating engine. Currently, the only supported templating
+engine is [Helm](https://helm.sh/), by relying the `helm template` command. See
+more details [here](https://docs.helm.sh/helm/#helm_template).
+
+> Note: Make sure that you have configured [artifact support](/setup/artifacts)
+> in Spinnaker first. All Helm charts are fetched/stored as artifacts in
+> Spinnaker. Read more in the [reference pages](/reference/artifacts).
+
+## Configure the "Bake (Manifest)" stage
+
+When configuring the "Bake (Manifest)" stage, you have the option to configure
+the following:
+
+* __The release name__
+
+  This is the Helm release name for this chart, and informs the name of the
```

Comment:
I think you can remove "This is..."

* "The Helm release name..."

* "The Help chart that you will..."

* "The parameters to the..."